lib/components/helpers/label.js

```diff
@@ -26,6 +26,12 @@ export default createReactClass({
     var config = this.props.config;
     var field = this.props.field;
     var fieldLabel = config.fieldLabel(field);
+    var fieldIsRequired = config.fieldIsRequired(field);
+    var requiredTextClassName = cx({
+      'required-text': fieldIsRequired,
+      'not-required-text': !fieldIsRequired,
+      'has-required-text': !!field.required_text,
```

Comment:
Note how we don't access the `field.required` property directly. That's a pattern to avoid binding the components directly to the fields and instead allow the `config` to override that. So, instead, we should add a method to the config so you can do `config.fieldRequiredText(field)` here.

For the `default-config` implementation, return `field.requiredText` just because Formatic uses camelCased properties. (It allows snake_case for helpText but nevermind that.)

But if we provide the same exact text for every field, the plugin on the Zapier could just be implemented as:

```js
config.fieldRequiredText = () => 'This field is required.';
```